esm/html/document.js

```diff
@@ -102,3 +103,53 @@ export class HTMLDocument extends Document {
     return element;
   }
 }
+
+/** @typedef {{ [WINDOW] : Object }} Context */
+/**
+ * @constructor
+ * @param {Context} context
+ */
+export class HTMLDocument extends _HTMLDocument {
+
+  constructor (context = {}) {
+    super();
+    const useState = (target, name) => {
+      return ((_0, _1={}) => {
+        const g = (name in _1) ? _1 : _0;
```

Comment:
this easily leaks the Object.prototype namespace to the window ... probably not a real-world concern, yet `hasOwnProperty` might be better